### PR TITLE
Fix a typo in the RestActionLoader

### LINKED ISSUE #1318: branch 1.7 is broken
due to commit e637ce6b7f3763b973b917ba29b00b928193c46a

https://github.com/FriendsOfSymfony/FOSRestBundle/blob/1.7/Routing/Loader/Reader/RestActionReader.php#L355

```php
Warning: is_subclass_of() expects at least 2 parameters, 1 given
```
This : 
```php
is_subclass_of($argumentClass, $className)
```

Instead of : 
```php
is_subclass_of($className)
```

### PR DESCRIPTION
**Pull request by Ener-Getick**
Fix #1318

### CODE REVIEW

**Comment by Ener-Getick** [comment]
@xabbuh @lsmith77 do you see anything else to change/add?

File: Tests/Fixtures/Controller/AnnotatedPrefixedController.php [hunk 1]
```diff
@@ -20,7 +21,7 @@
  */
 class AnnotatedPrefixedController extends Controller
 {
-    public function getSomethingAction()
+    public function getSomethingAction(Request $request)
```
Comment (xabbuh) [inline Tests/Fixtures/Controller/AnnotatedPrefixedController.php]: What is this change needed for?

File: Tests/Fixtures/Controller/ArticleController.php [hunk 2]
```diff
@@ -23,7 +24,7 @@ public function optionsAction()
 
  // [OPTIONS] /articles
 
-    public function cgetAction()
+    public function cgetAction(ConstraintViolationList $errors)
```
Comment (xabbuh) [inline Tests/Fixtures/Controller/ArticleController.php]: And this?

File: Tests/Fixtures/Controller/AnnotatedPrefixedController.php (hunk 1 above)
Comment (Ener-Getick) [inline Tests/Fixtures/Controller/AnnotatedPrefixedController.php]: To check that the routing is working and that this type hints do not affect the routes.

**Comment by Ener-Getick** [comment]
Is it ok for you?

**Comment by lsmith77** [comment]
yeah

Status: Reviewed

**Comment by Ener-Getick** [comment]
Cool thank you :-)

(My implementation of carsonbot seems to be down, i'll investigate...)

**Pull request merged**